easybuild/easyconfigs/c/cairo/cairo-1.14.10-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'cairo'
+version = '1.14.10'
```

Comment:
@verdurin We usually just pick a version (e.g. latest here), but sometimes we need to bump the `GLib` version without having a more recent `cairo` available, and then we add a `versionsuffix` to discriminate from the existing `cairo` with a previous `GLib` version.

We also do this in other situations (e.g. `libpng` & `freetype`).